semiphemeral/settings.py

```diff
@@ -54,6 +54,7 @@ def load(self):
 
     def save(self):
         with open(self.filename, "w") as f:
+            os.chmod(self.filename, 0o0600)
```

Comment:
Several thoughts:

- I think more than just settings should be protected; the database too, since it may have DMs, etc.
- setting a safe umask at the start of execution seems better.
- this fix is technically racey; it needs to use an opener arg to `open` to set the mode with `os.open(..., 0o600)` so the file is never readable.